frontend/src/app/signUp/IdentityVerification/PersonalDetailsForm.tsx

```diff
@@ -96,6 +98,32 @@ const PersonalDetailsForm = ({
     )?.focus();
   };
 
+  /**
+   * Focus on the first input with errors
+   */
+  useEffect(() => {
```

Comment:
I saw this method for focusing error on another component as well - is it potentially worth extracting the logic into a helper method, or do you just think that would make everything more complicated? I'm specifically talking about the body of the `if` statement.

Reply:
Good idea!